Phase03/Phase03-FullTextSearchRefactor/UI/UserCriteria.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Phase03_FullTextSearchRefactor.UI;
+
+internal class UserCriteria
```

Comment:
make this object immutable (why?)

Reply:
solved.

Reply:
because when the fields in userCriteria are initialized by user, it should not be changed afterward.